brave/browser/guest_view/tab_view/tab_view_guest.cc

```diff
@@ -171,7 +173,7 @@ void TabViewGuest::Load() {
   auto tab_helper = extensions::TabHelper::FromWebContents(web_contents());
   if (tab_helper && !tab_helper->IsDiscarded()) {
     if (!web_contents()->HasOpener())
-      api_web_contents_->ResumeLoadingCreatedWebContents();
+      web_contents()->ResumeLoadingCreatedWebContents();
```

Comment:
this doesn't actually change anything, but there was no reason to call it through api_web_contents_